### PR TITLE
Add project for repackaging Jersey

### PR DESCRIPTION
**Pull request by benjaminr-ps**

### CODE REVIEW

File: org.eclipse.lyo.repackaging.jersey/pom.xml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.eclipse.lyo.oslc4j.core</groupId>
+	<artifactId>repackaging.jersey</artifactId>
+	<version>2.26</version>
```
Comment (berezovskyi) [inline org.eclipse.lyo.repackaging.jersey/pom.xml]: do we want to use Lyo versioning here instead?

**Comment by yashkhatri** [comment]
@berezovskyi I will verify it this week.

File: org.eclipse.lyo.repackaging.jersey/pom.xml (hunk 1 above)
Comment (yashkhatri) [reply org.eclipse.lyo.repackaging.jersey/pom.xml]: I think, the Jersey version required here is 2.26. So if we change it here to the Lyo version, make sure to set jersey version correctly.

File: org.eclipse.lyo.repackaging.jersey/pom.xml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.eclipse.lyo.oslc4j.core</groupId>
+	<artifactId>repackaging.jersey</artifactId>
+	<version>2.26</version>
+
+	<properties>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<jersey.version>${project.version}</jersey.version>
```
Comment (yashkhatri) [inline org.eclipse.lyo.repackaging.jersey/pom.xml]: 2.26, if we change the project version above to the Lyo version.

Comment (berezovskyi) [reply org.eclipse.lyo.repackaging.jersey/pom.xml]: Exactly. @jadelkhoury @jamsden do you think the repackaged jersey should have a "lyo" version of 2.4.0 in the upcoming release or 2.26 of the jersey artifacts that are being repackaged here?

**Comment by yashkhatri** [comment]
+1 verified.

File: org.eclipse.lyo.repackaging.jersey/pom.xml (hunk 2 above)
Comment (jadelkhoury) [reply org.eclipse.lyo.repackaging.jersey/pom.xml]: no idea, but I'll try to say something smart.
This looks like a pom file with a group id "org.eclipse.lyo.oslc4j.core". so spontaneous reaction is to follow the lyo version.
But if this only about repackaging Jersey, with no Lyo-related code, then 2.26 makes sense.
There you go: No wise answer really. (as usual, a long phone chat might resolve it quicker)

**Comment by berezovskyi** [comment]
needs to be SNAPSHOT until the release

**Comment by berezovskyi** [comment]
Should move to 2.4.0 so that the release process is the same as for the rest of the modules (drop the SNAPSHOT). Otherwise would be impossible (the Eclipse Maven repo would reject repeated depoys of an already released 2.26 version) to keep it in the same repo under the same parent POM. @jadelkhoury FYI (your argument is still valid and 2.26 is better semantically and needs to be updated only when an underlying dependency is updated, but this would need a separate repo).

**Comment by berezovskyi** [comment]
@benjaminr-ps did you allow edits from maintainers?

**Comment by benjaminr-ps** [comment]
Yes, it is fine to be edited

**Comment by berezovskyi** [comment]
@benjaminr-ps could you please ensure the checkbox is selected? https://stackoverflow.com/questions/20928727/adding-commits-to-another-persons-pull-request-on-github

**Comment by yashkhatri** [comment]
@berezovskyi Now, it is accessible. So, I have pushed the commit here. I didn't see this comment of yours and Benjamin's before so I created a new PR. I will close the other PR now.

**Pull request merged**